src/Eccube/Controller/Admin/Product/CsvImportController.php

```diff
@@ -481,6 +486,19 @@ public function csvCategory(Application $app, Request $request)
 
                         }
 
+                        // Rank
+                        if (Str::isBlank($row['表示ランク'])) {
+                            if (!$Category->getRank()) {
```

Comment:
can duplicate rank in case:
1,2,3,4 items null, next items is = max in DB + 1 
==> duplicated with item 1